app/Http/Controllers/ChopsController.php

```diff
@@ -16,107 +17,109 @@
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\Auth;
 use Illuminate\Support\Facades\Redirect;
+use Illuminate\Support\Facades\Input as Input;
+use Validator;
+use Response;
 use Cloudder;
 
+\Cloudinary::config(array(
+    "cloud_name" => "chopbox"
+));
```

Comment:
It supplies name of Cloudinary account for this app.
Even though this has been supplied in the env file, Cloudinary cl_image_tag() for retrieving and manipulating the images on view home.blade.php requests for it.